### ISSUE TITLE
Add Development menu to Admin Bar

### ISSUE THREAD

**Issue by ghost** [backdrop-contrib/devel#7]
In Drupal, the Development links were accessible from the Admin Menu (specifically the Home menu item). In Backdrop, there's a separate Development menu, but it's not visible by default and not accessible from the Admin Bar.

Is there a way of making these links accessible from the Admin Bar, or can we add that as a feature request here?

**Comment by quicksketch** [comment]
> Is there a way of making these links accessible from the Admin Bar, or can we add that as a feature request here?

Yes please! It makes sense to be here in Devel module. I'm guessing that in D7, Admin Menu provided the integration and when we did our port to core, we dropped all contrib module integration. Or it's possible that the admin_menu code is still here within Devel but just isn't updated for admin_bar yet.

**Comment by sirkitree** [comment]
I'd love to see the `user/switch` stuff in the admin menu too, if not adding to the user indicator that exists, somewhere close by.

**Comment by quicksketch** [comment]
I restructured the way menu items work and removed a bunch of obsolete options (cache clear, cron run, menu rebuild) that admin bar replaces in https://github.com/backdrop-contrib/devel/commit/1c0145b3d4fcc45998f0161573edcbeac2cf6406. For now I've only exposed the existing menu items under a single top-level menu item (Development) and deleted the custom "devel" menu. Adding switch users under the user list or other features we can add in a new issue.
